mqgo/src/meqa/mqgen/main.go

```diff
@@ -29,19 +29,34 @@ func main() {
 	swaggerFile := flag.String("s", swaggerJSONFile, "the swagger.yml file location")
 	algorithm := flag.String("a", "all", "the algorithm - simple, object, path, all")
 	verbose := flag.Bool("v", false, "turn on verbose mode")
+	whitelistFile := flag.String("w", "", "the whitelist.txt file location")
 
 	flag.Parse()
-	run(meqaPath, swaggerFile, algorithm, verbose)
+	run(meqaPath, swaggerFile, algorithm, verbose, whitelistFile)
 }
 
-func run(meqaPath *string, swaggerFile *string, algorithm *string, verbose *bool) {
+func run(meqaPath *string, swaggerFile *string, algorithm *string, verbose *bool, whitelistFile *string) {
 	mqutil.Verbose = *verbose
 
 	swaggerJsonPath := *swaggerFile
 	if fi, err := os.Stat(swaggerJsonPath); os.IsNotExist(err) || fi.Mode().IsDir() {
 		fmt.Printf("Can't load swagger file at the following location %s", swaggerJsonPath)
 		os.Exit(1)
 	}
+	whitelistPath := *whitelistFile
+	var whitelist map[string]bool
+	if len(whitelistPath) > 0 {
+		if fi, err := os.Stat(whitelistPath); os.IsNotExist(err) || fi.Mode().IsDir() {
+			fmt.Printf("Can't load whitelist file at the following location %s", whitelistPath)
```

Comment:
Here we have used fmt.Printf and in line 56 we are using mqutil.Logger.Printf, any reason for using two different approaches for printing messages.

Reply:
`fmt.Print` prints to the `stdout` whereas `mqutil.Logger` can print both to `stdout` as well as a specified log file.